ui/jobs/buff_tracker.ts

```diff
@@ -1,10 +1,54 @@
+import { BaseRegExp, Matches } from '../../types/trigger';
+import { IJobsOptions } from './types';
+
+import WidgetList from '../../resources/widget_list';
 import EffectId from '../../resources/effect_id';
+import { AbilityParams, GainsEffectParams, LosesEffectParams } from '../../resources/netregexes';
```

Comment:
This is just a question for my own curiosity because it came up in `ui/radar/radar.ts` that I'm working on, but what happens when you need to import both default and non-default parameters? Is that possible?

Reply:
It is just my guess, and I have not verified it: I think that you can write something like,
```typescript
import { AbilityParams } from 'path/to/netregexes';
type NonDefaultParameters = 'para1' | 'para2' | 'para3';
type CustomizedMatches = Matches<BaseRegExp<AbilityParams | NonDefaultParameters>>;
```

In my opinion, the typing of `Matches` and `BaseRegExp` is not good for these usages. Perhaps I and @Trim21 would rewrite the typing later.

Reply:
Oh oh, sorry.  I think I didn't explain myself well.  I didn't realize until now that you could do `import NetRegexes, { AbilityParams } from './netregexes';`.  Sorry I meant default/non-default *exports* and not *parameters*.  I misspoke.

Reply:
Ah, yes! You could do that! ^_~